packages/tatum-ada/src/ledger/vc.ts

```diff
@@ -4,4 +4,4 @@ export {
   updateVirtualCurrency,
   mintVirtualCurrency,
   revokeVirtualCurrency,
-} from '@tatumio/tatum-core'
+} from '@tatumio/tatum-ledger'
```

Comment:
Should I remove these re-exports or is that necessary to have them here in specific chain module?